utils/ci/run_in_gha.sh

```diff
@@ -80,6 +80,16 @@ echo ::group::Setting up daos_server_helper
 ./utils/setup_daos_server_helper.sh
 echo ::endgroup::
 
+echo ::group::Tar /opt/daos et /home/daos/daos
+echo backing-up /opt/daos in /tmp/${DOCKER_DISTRO}_${BASE_RUNNER}_opt_daos.tgz
+tar czf /tmp/${DOCKER_DISTRO}_${BASE_RUNNER}_opt_daos.tgz /opt/daos
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]